### DISCUSSION TITLE
Make Nextcloud and website talk to each other (webdav?)

### DISCUSSION THREAD

**Discussion by Battleman** [clicepfl/clic-website#235] [Ideas]
At the moment, content on Nextcloud (in particular images and videos) are totally hidden from the website. In particular, if we wish to display content on the website it must be either copied (and tracked) using github, or forcefully copied and untracked (because too heavy c.f. Stallman video).

It would be better to allow the website/CMS to read from the Nextcloud, one way or another. This will allow to track media content with nextcloud and not github anymore, and have more dynamic content (e.g. have a dynamic gallery, and not some hardcoded images).

Important to remember that there are different unix users for nextcloud and web (good security practice, but make it harder)

Some possible ways:
* Symlink of nextcloud in website, with some common unix group
* Use webdav and internal routes redirection to a specific nextcloud folder
* ??

**Comment by mochiruntime** [comment]
Let's move this to discussions, we'll create concrete issues once we figure out which implementations we need
